composer.json

```diff
@@ -1,17 +1,22 @@
 {
     "name": "heureka/overenozakazniky",
-    "description": "Heureka overeno service API",
+    "description": "Heureka overeno zakazniky service API",
     "authors": [
         {
             "name": "Heureka.cz",
             "email": "podpora@heureka.cz"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "php": ">=5.1.0"
+        "php": ">=5.4.0"
```

Comment:
tyjo, tohohle se trochu bojim, verim ze spousta lidi jeste pouziva php 5.3 protoze si to jednou na hostingu nastavili, funguje jim to a tak nemaj duvod si to menit